packages/flutter/lib/src/material/ink_ripple.dart

```diff
@@ -120,11 +121,13 @@ class InkRipple extends InteractiveInkFeature {
     ShapeBorder customBorder,
     double radius,
     VoidCallback onRemoved,
+    TextDirection textDirection,
```

Comment:
This class can fail if textDirection is null - why isn't this parameter @required and asserted to be non-null?

Reply:
Added assert to constructor and required to create/constructor